tycho-sbom/src/main/java/org/eclipse/tycho/sbom/TychoProjectDependenciesConverter.java

```diff
@@ -57,6 +58,8 @@ public class TychoProjectDependenciesConverter extends DefaultProjectDependencie
 	@Inject
 	private MavenProjectDependencyProcessor dependencyProcessor;
 
+	private final Map<IInstallableUnit, List<String>> bomRepresentations = new HashMap<>();
```

Comment:
Better use a `ConcurentHashMap` to support parallel builds

Reply:
A fair point. I just double-checked and the map is indeed shared across the entire build.